sklearn/datasets/value_dropper.py

```diff
@@ -0,0 +1,300 @@
+# Author : Raghav RV <rvraghav93@gmail.com>
+#
+# Licence : BSD 3 clause
+
+import numpy as np
+import numbers
+
+from sklearn.utils import check_array
+from sklearn.utils import check_random_state
+from sklearn.utils.multiclass import type_of_target
+
+from sklearn.base import TransformerMixin
+from sklearn.preprocessing import LabelEncoder
+
+
+__all__ = ["ValueDropper"]
+
+
+class ValueDropper(TransformerMixin):
```

Comment:
It was initially a function and later made as a transformer... I did so assuming it would be useful in a pipeline? Do you feel it's simpler to have it as a function?